gulp/tasks/sass.js

```diff
@@ -17,9 +17,8 @@ module.exports = function(gulp, options) {
       browserSync.notify('Compiling Sass');
     }
 
-    return sass(config.src, config.options)
-      .pipe(plumber({errorHandler: options.onError}))
-      .on('error', options.onError)
+    return gulp.src(config.src)
+      .pipe(sass(config.options).on('error', sass.logError))
```

Comment:
This changes how error handling works - previously, `errorHandler` from gulpfile.js would get called, respecting `--fail`, etc. Sass logError does something else, right? Might make sense to keep using `options.onError`?